meshers/blocky/voxel.cpp

```diff
@@ -94,6 +94,10 @@ void Voxel::set_color(Color color) {
 	_color = color;
 }
 
+void Voxel::set_bake_tangents(bool bt) {
+	_bake_tangents = bt;
```

Comment:
I think the flag doesnt have to exist on `Voxel`, it may be a property of `VoxelLibrary` only. Then it can get passed to `Voxel::bake` functions as an argument.

Reply:
Switched to doing it that way and being on by default.